code/modules/mob/living/simple_animal/friendly/dog.dm

```diff
@@ -1,542 +1,568 @@
-//Corgi

```

Comment:
This looks like a line ending issue thats causing git to report every line as changed. The file should be CRLF, so you'll need to convert it back to that from likely LF that it got automagically changed it, or vice versa if my memory is wrong. You can do it with Notepad++.